README.md

```diff
@@ -22,18 +22,18 @@ composer require ipinfo/ipinfo
 
 #### Quick Start
 
-```
->>>use ipinfo\ipinfo\IPinfo;
+```php
+require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
I think we usually don't write this at the top of PHP snippets?

Reply:
If we remove this snippet, we should notify user that loading the required dependencies before using the `IPinfo` class.

Otherwise it will cause the problem about this [issue](https://github.com/ipinfo/php/issues/39).

As we can see above issue, it doesn't use any related dependencies loading before calling the `IPinfo` class.

Reply:
Alright!